src/gitgraph.js

```diff
@@ -1388,6 +1439,7 @@
    * @private
    */
   function _drawTextBG ( context, x, y, text, color, font, angle, useStroke ) {
+
```

Comment:
Sorry, it's a habit of mine to add more whitespace when working on a function. Scheduled for removal.